crates/findex/src/pyo3/mod.rs

```diff
@@ -10,14 +9,119 @@ macro_rules! pyo3_unwrap {
     };
 }
 
+/// Implements the basic functionalities of keyword and location in python.
+///
+/// # Parameters
+///
+/// - `type_name`   : name of the key type
+macro_rules! impl_python_byte {
+    ($py_type:ty, $rust_type:ty, $name:tt) => {
```

Comment:
Would a literal be okay here?
```suggestion
    ($py_type:ty, $rust_type:ty, $name:literal) => {
```